main.js

```diff
@@ -0,0 +1,35 @@
+var name = $('#postname')
+var post = $('#posttext')
+
+var deletePost = $('<button type="button" class="btn-close" aria-label="Close"></button>')
+
+
+
+$("#postbtn").on('click', function () {
+  console.log('click')
+
+
+  var deletePost = $('<button type="button" class="btn-close" aria-label="Close"></button>').addClass("deletebtn")
+
+  var name = $("#postname").val();
+  var post = $("#posttext").val();
+  var box = $('.newname')
+  var delBtn = $('.btn-close')
+  var postFunc = function () {
```

Comment:
Would prefer for this function to be defined outside of the scope of the on click callback for the post button. Also, the naming could be more semantic here.